src/nuclio/projects/project/functions/version/version-configuration/tabs/version-configuration-annotations/version-configuration-annotations.component.js

```diff
@@ -31,6 +32,55 @@
             $i18next.t('functions:TOOLTIP.ANNOTATIONS.HEAD', {lng: lng}) + '</a> ' +
             $i18next.t('functions:TOOLTIP.ANNOTATIONS.REST', {lng: lng});
 
+        ctrl.keyTooltip = $i18next.t('functions:TOOLTIP.PREFIXED_NAME', {
+            lng: lng,
+            name: $i18next.t('functions:TOOLTIP.ANNOTATION', {lng: lng})
+        });
+        ctrl.validationRules = {
+            key: [
+                {
+                    name: 'nameValidCharacters',
+                    label: '[' + $i18next.t('common:NAME', {lng: lng}) + '] ' + $i18next.t('functions:VALID_CHARACTERS', {lng: lng}) + ': a–z, A–Z, 0–9, -, _, .',
+                    pattern: /^([^\/]*\/)?[\w-.]+$/
+                },
+                {
+                    name: 'nameBeginEnd',
+                    label: '[' + $i18next.t('common:NAME', {lng: lng}) + '] ' + $i18next.t('functions:BEGIN_END_WITH_ALPHANUMERIC', {lng: lng}),
+                    pattern: validateNameBeginEnd
+                },
+                {
+                    name: 'nameMaxLength',
+                    label: '[' + $i18next.t('common:NAME', {lng: lng}) + '] ' + $i18next.t('functions:MAX_LENGTH_CHARACTERS', {lng: lng, count: 63}),
+                    pattern: /^([^\/]*\/)?[\S\s]{1,63}$/
+                },
+                {
+                    name: 'prefixValidCharacters',
+                    label: '[' + $i18next.t('function:PREFIX', {lng: lng}) + '] ' + $i18next.t('functions:VALID_CHARACTERS', {lng: lng}) + ': a–z, 0–9, -, .',
```

Comment:
Bug: UI shows “PREFIX” in uppercase, because it doesn’t find `i18next`’s namespace `function`.
```suggestion
                    label: '[' + $i18next.t('functions:PREFIX', {lng: lng}) + '] ' + $i18next.t('functions:VALID_CHARACTERS', {lng: lng}) + ': a–z, 0–9, -, .',
```
Please fix the same in the rest of the file where you wrote by mistake `function:` instead of `functions:`